languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -14933,6 +14933,11 @@ though, and has since been slightly modified)-->
       </rule>
     </rulegroup>
     <rule id="ACCORD_CENTAINE_DE" name="centaine de">
+      <antipattern>
+          <token inflected="yes">série</token>
+          <token>de</token>
+          <token>bande dessinée</token>
+      </antipattern>
```

Comment:
We need 2 tokens: 
```
<token>bande</token>
<token>dessinée</token>
```
And a test sentence: `<example>série de bande dessinée</example>`

Reply:
If you want to make a change to a pull request, for example in this case to split "bande dessinée" into two tokens, and to add an `<example>` to test if your AP works: just check out the branch on which you made the original PR (in this case, `RM-2`), make the changes, commit them and push to the same branch. You PR will be updated automatically.